www/source/blog/2017-06-07-Triage-and-Contributions.html.md

```diff
@@ -0,0 +1,30 @@
+
+Many of us have probably watched other FOSS communities adopt different strategies to minimize the barrier to contribution and the habitat project is not an exception. One of our project beliefs is that building humane software requires our processes, our code of conduct, our Governance and our developer tooling to reflect that humanity. Which, is why we're opting to make some simple governance-ish changes that we hope to be the first steps towards minimizing the barrier to contribution in the [habitat.sh codebase](https://github.com/habitat-sh/habitat).
+
+## Tag Categorization
+The first change is relatively low impact but if you spend time tracking our development in GitHub then you might have noticed it. We've opted to change the system by which we are tagging and tracking our issues. "Wow... what a build up for a such a trivial change there, Ian." I know, I know but, hear me out. First lets look at how this is going to work now and then we can discuss where we hope to lead this forward. We've broken all of our GitHub Labels into groups which I'll define here:
+
+| *TAGS Groups*                     |               Meaning                             |
+|:----------------------------------|:--------------------------------------------------|
+| *AREA TAGS* Prefix "A-"           | What part of the codebase does the issue refer to.|
+| *CATEGORIZATION TAGS* Prefix "C-" | What type of work does the issue refer to.        |
+| *EFFORT TAGS* Prefix "E-"         | Should a contributor undertake, what is the estimated effort level. |
+| *LANGUAGE TAGS* Prefix "L-"       | Pretty straightforward right? What language the work will involve.  |
+| *STATUS TAGS* Prefix "S-"         | The current status of an issue or pull request.   |
+
+Obviously these specific labels might change in the future and for a full breakdown of what each individual tag means you can check out the [Label Names and Definitions](https://github.com/habitat-sh/habitat/wiki/Label-Names-and-Definitions) post in the [wiki](https://github.com/habitat-sh/habitat/wiki) (side note on the wiki: this is another item we plan on fleshing out with content as we continue forward with the project!).
+
+The goal of these changes is to quickly and efficiently bubble up snippets of work for new and recurring project contributors. We hope that during triage we can start estimating level of effort, providing theoretical starting points, and flagging the piece of the project an issue refers to in order to give our community a really low friction way to get involved in the hacking! But we don't plan to stop here. We are always looking for ways to expose and encourage community members to get involved and we feel like taking this step first really enables us to build out some community work we'd like to be doing like social-coding, mentoring new contributors, and providing better metrics and tooling around the contribution process.
```

Comment:
This would be cool but I'm not certain if we're quite ready to provide starters like that just yet. I'm planning to spend some time digging into previously triaged issues to try and estimate level of effort etc. But as we weren't doing this before we're going to have to retroactively build up the label/descriptor base.

Would you prefer we wait to publish the blog post until we can include that content?